src/components/visualizations/chart/highcharts/customConfiguration.ts

```diff
@@ -325,12 +325,31 @@ function level2LabelsFormatter(config?: IChartConfig) {
     return `${get(this, 'point.name')} (${formatLabel(get(this, 'point.value'), get(this, 'point.format'), config)})`;
 }
 
+function isNullOrUndefined(value: any) {
```

Comment:
maybe `_.isNil(value)` from lodash does the same? please check

Reply:
👍 fixed